autoload/ctrlp/line.vim

```diff
@@ -50,11 +51,17 @@ fu! ctrlp#line#init(bufnr)
 	retu lines
 endf
 
-fu! ctrlp#line#accept(mode, str)
-	let info = matchlist(a:str, '\t|[^|]\+|\(\d\+\):\(\d\+\)|$')
+fu! ctrlp#line#accept(dict)
+	let mode = a:dict['action']
+	let str = a:dict['line']
+	let input = a:dict['input']
+	let info = matchlist(str, '\t|[^|]\+|\(\d\+\):\(\d\+\)|$')
 	let bufnr = str2nr(get(info, 1))
 	if bufnr
-		cal ctrlp#acceptfile(a:mode, bufnr, get(info, 2))
+		cal ctrlp#acceptfile(mode, bufnr, get(info, 2))
+		let @/ = input
```

Comment:
This produces an error if a line is selected before anything is typed (e.g. just press <Enter> after CtrlPLine is opened). PR #413 fixes it.